player/src/sc/player2021/Starter.java

```diff
@@ -34,6 +38,8 @@ public Starter(String host, int port, String reservation) throws Exception {
   }
 
   public static void main(String[] args) {
+    GamePlugin.getXStream();
```

Comment:
doesn't the player load the plugin at some point? That should make this superfluous.

Reply:
It should, but it doesn't. This is a workaround to force the loading and prevent the linked issue. Any idea why it doesn't work without?

Reply:
We had the issue with Colors and Teams already, I still haven't figured out why.
Maybe Java doesn't load Kotlin code automatically and can't figure out the need here.
Maybe it works with a Kotlin client, I'll look into it.

Reply:
fixed